<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPICViewer</class>
 <widget class="QMainWindow" name="VPICViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VPIC Data Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="TreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="PlotConfig">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Plot Configuration</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>7</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="horizontalSpacing">
                <number>7</number>
               </property>
               <property name="verticalSpacing">
                <number>7</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="XAxisLabel">
                 <property name="text">
                  <string>X-Axis</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="XAxis"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="YAxisLabel">
                 <property name="text">
                  <string>Y-Axis</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="YAxis"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="Z1Label">
                 <property name="text">
                  <string>Z1-Axis</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="Z1Slider">
                 <property name="tracking">
                  <bool>false</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="Z2Label">
                 <property name="text">
                  <string>Z2-Axis</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSlider" name="Z2Slider">
                 <property name="tracking">
                  <bool>false</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="PlotFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenDir"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>VPICViewer</receiver>
   <slot>file_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenDir</sender>
   <signal>triggered()</signal>
   <receiver>VPICViewer</receiver>
   <slot>dir_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XAxis</sender>
   <signal>activated(QString)</signal>
   <receiver>VPICViewer</receiver>
   <slot>update_axes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YAxis</sender>
   <signal>activated(QString)</signal>
   <receiver>VPICViewer</receiver>
   <slot>update_axes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>VPICViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Z1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VPICViewer</receiver>
   <slot>update_sliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Z2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VPICViewer</receiver>
   <slot>update_sliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>VPICViewer</receiver>
   <slot>change_dataset(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>file_dialog()</slot>
  <slot>dir_dialog()</slot>
  <slot>change_dataset(QModelIndex)</slot>
  <slot>update_axes()</slot>
  <slot>update_sliders(int)</slot>
 </slots>
</ui>
